src/FluffySpoon.AspNet.LetsEncrypt/ChallengeType.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FluffySpoon.AspNet.LetsEncrypt
+{
+	[Flags]
+	public enum ChallengeType
+	{
+		Http01 = 1,
```

Comment:
Why `Http01` and not `Http`?

Reply:
It matches the ACME protocol names, which are numbered to allow versioning i.e. dns-01 and http-01. This allows for implementation of future protocols or different versions of the existing ones. It's also consistent with the underlying Certes library.

Reply:
There's also an existing protocol (TLS ALPN) which could be supported now, if someone wanted to implement it. I have no need for it, nor any way to test it.

Reply:
Fair enough :+1: